MiniApp/Classes/core/Display/MiniAppWebView.swift

```diff
@@ -14,25 +14,38 @@ internal class MiniAppWebView: WKWebView {
 
     convenience init(miniAppId: String, versionId: String, queryParams: String? = nil) {
         let schemeName = Constants.miniAppSchemePrefix + miniAppId
-        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName + MiniAppWebView.getQueryParams(queryParams: queryParams))!)
         let config = MiniAppWebView.defaultConfig()
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
         self.init(frame: .zero, configuration: config)
-        commonInit(urlRequest: urlRequest)
+        commonInit(urlRequest: Self.getURLRequest(miniAppId: miniAppId, schemeName: schemeName, queryParams: queryParams))
     }
 
     convenience init(miniAppURL: URL, queryParams: String? = nil) {
-        let urlWithQueryParam = miniAppURL.appendingPathComponent(MiniAppWebView.getQueryParams(queryParams: queryParams))
-        let urlRequest = URLRequest(url: urlWithQueryParam)
+        let urlRequest = URLRequest(url: Self.getURL(miniAppURL: miniAppURL, queryParams: queryParams))
```

Comment:
Actually, I don't think we need to append the query params to mini apps loaded by a URL. I think Android doesn't append params when loaded by URL, is that right @khairul-alam-licon 

If they want to add query params in this case, they can just add them directly to the URL they are loading.